CHANGELOG.md

```diff
@@ -13,6 +13,10 @@ Our versioning strategy is as follows:
 
 ### 🎉 New Features & Improvements
 
+* `[templates/nextjs][sitecore-jss-react]` Enable client-only BYOC component imports. Client-only components can be imported through src/byoc-imports/index.client.ts. Hybrid (server render with client hydration)components can be imported through src/byoc-imports/index.ts ([#1628](https://github.com/Sitecore/jss/pull/1628))
```

Comment:
We should prob update this to mention index.hybrid.ts instead of index.ts

Reply:
True, but I was postponing this until the very end to outline all changes, after all comments are resolved.
Is this the last thing to do? If so - I can update changelog and merge